actionview/lib/action_view/helpers/debug_helper.rb

```diff
@@ -5,32 +5,64 @@
 module ActionView
   # = Action View Debug Helper
   #
-  # Provides a set of methods for making it easier to debug Rails objects.
+  # Provides a method to debug objects.
   module Helpers # :nodoc:
     module DebugHelper
       include TagHelper
 
-      # Returns a YAML representation of +object+ wrapped with <pre> and </pre>.
+      # Returns a YAML representation of +object+ wrapped with <pre> tags.
       # If the object cannot be converted to YAML using +to_yaml+, +inspect+ will be called instead.
       # Useful for inspecting an object at the time of rendering.
       #
       #   @user = User.new({ username: 'testing', password: 'xyz', age: 42})
       #   debug(@user)
+      #
       #   # =>
-      #   <pre class='debug_dump'>--- !ruby/object:User
-      #   attributes:
-      #     updated_at:
-      #     username: testing
-      #     age: 42
-      #     password: xyz
-      #     created_at:
+      #   <pre class="debug_dump">--- !ruby/object:User
```

Comment:
This seems too verbose to me. Original example was simple enough to get clear picture what's happening in 8 lines.

Reply:
Reasonable. I agree.